syllabus/readme.md

```diff
@@ -54,16 +54,14 @@ Each course element (1-12) is a four hour session, consisting of a 2hr lecture a
 | 39 |  |  |  |  |
 | 40 | 3 | Document representations | Classification from representations | *Baroni et al., 2014* (spend the extra time to go to the NLP worksop) |
 | 41 | 4 | Neural Networks | Neural networks with `pytorch` | *Nielsen 2019, Chapter 1*, 3Blue1Brown Videos (ch.[1](https://www.youtube.com/watch?v=aircAruvnKk&list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi), [2](https://www.youtube.com/watch?v=IHZwWFHWa-w&list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi&index=2)) |
-| 43 | 5 | Contextual Embeddings I | Comparing contextual vectors | *Vaswani et al 2017*; *Lindsay et al 2020* 3Blue1Brown Videos (ch. [5](https://www.youtube.com/watch?v=wjZofJX0v4M&list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi&index=5), [6](https://www.youtube.com/watch?v=eMlx5fFNoYc&list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi&index=6), optionally [7](https://www.youtube.com/watch?v=9-Jl0dxWQs8&list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi&index=7)) |
-| 44 | 6 | Too be added |  |  |
-
-<!-- | 44 | 6 | Contextual Embeddings II |  | Too be announced |
-| 45 | 7 | Transfer learning | Exploring BERT (NER) | *Devlin et al., 2019*; *Rogers et al., 2020* |
-| 46 | 8 | Language generation I | Prompt engineering | *Brown et al 2020*; *Raffel et al 2019* |
-| 47 | 9 | Language generation II | Training a model with instruction tuning | *Wei et al., 2021*; *Ouyang et al., 2022* |
-| 48 | 12 | Ethics and social impact | Project development | *Bender et al., 2021*; *Mitchell et al., 2019* |
-| 49 | 11 | LLMs and Cognition | Discussion | *group-specific readings* |
-| 50 | 10 | Project presentations | Project feedback | Prepare projects | -->
+| 43 | 5 | Contextual Embeddings I, Attention | Comparing contextual vectors | *Vaswani et al 2017*; *Lindsay et al 2020* 3Blue1Brown Videos (ch. [5](https://www.youtube.com/watch?v=wjZofJX0v4M&list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi&index=5), [6](https://www.youtube.com/watch?v=eMlx5fFNoYc&list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi&index=6), optionally [7](https://www.youtube.com/watch?v=9-Jl0dxWQs8&list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi&index=7)) |
+| 44 | 6 | Contextual Embeddings II, Language generation | Prompt engineering  | Brown et al 2020; Raffel et al 2019 |
+| 45 | 7 | Contextual Embeddings III, Transfer learning and Probing | Probing  | Devlin et al., 2019; Rogers et al., 2020; Hewitt, 2019 |
+| 46 | 8 | Instruction tuning | Training a model with instruction tuning  | *Wei et al., 2021*; *Ouyang et al., 2022* |
+| 47 | 9 | Ethics and social impact   |  |  *Bender et al., 2021*; *Mitchell et al., 2019* |
+| 48 | 10 | LLMs and Cognition  | Project development and feedback  | *group-specific readings* |
+| 49 | 11 | Evaluation  | Evaluating NLP Systems  | Wang et al., 2018; Zheng et al., 2023; Chiang et al., 2024; Optional:  Glockner et al., 2018 |
+| 50 | 12 | Project presentation |  Project development  |   |
```

Comment:
@jankounchained will it be possible for you to take this lecture (I am out of country)